projects/core/src/product/store/product-state.ts

```diff
@@ -15,13 +20,13 @@ export interface ProductsState {
 }
 
 export interface ProductsSearchState {
-  results: any;
-  suggestions: any[];
-  auxResults: any;
+  results: ProductList;
+  suggestions: Suggestion[];
+  auxResults: ProductList;
   loading: boolean;
 }
 
 export interface ProductReviewsState {
   productCode: string;
-  list: any[];
+  list: ReviewList;
```

Comment:
I understand what you are thinking but with our current implementation this is a Review[]. To make it a ReviewList you would have to make a few other changes and I think it's easier to make this a review and change the few other types that will also have to change.